src/rules/no-export-load-in-svelte-module-in-kit-pages.ts

```diff
@@ -0,0 +1,58 @@
+import type * as ESTree from "estree"
+import { createRule } from "../utils"
+import fs from "fs"
+
+const hasSvelteKit = (() => {
+  try {
+    const packageJson = JSON.parse(fs.readFileSync("./package.json", "utf8"))
+    // Hack: CI removes `@sveltejs/kit` and it returns false and test failed.
+    // So always it returns true if it runs on the package.
+    if (packageJson.name === "eslint-plugin-svelte") return true
+    return Boolean(
+      packageJson.dependencies["@sveltejs/kit"] ??
+        packageJson.devDependencies["@sveltejs/kit"],
+    )
+  } catch (_e) {
+    return false
+  }
+})()
```

Comment:
If a user doesn't use SvelteKit, I don't want to show an error, so I added to check package.json.
Is it the correct way?
And in the CI removes SvelteKit, so for the test, I added chack logic with `packageJson.name`
I think this is a bad idea but I don't have a better idea.

If correct, I will move it to utils for other SvelteKit-related rules.